radio/src/targets/common/arm/stm32/usbd_desc.c

```diff
@@ -68,7 +68,7 @@
 #define USBD_VID_PID_CODES                  0x1209    // https://pid.codes
 
 #define USBD_LANGID_STRING                  0x409
-#define USBD_MANUFACTURER_STRING            "OpenTX"
+#define USBD_MANUFACTURER_STRING            "EdgeTX"
```

Comment:
This probably needs to stay OpenTX for now, as it seems some simulators/games are completely dumb, and expect it to be OpenTX to detect the handset. i.e. DCL on PS4 - #4580

Until/unless we can change this at runtime, as a compatability/legacy option.

Reply:
Now this is really fun :-D Ok, then let's keep it as is...

Reply:
I know... daft, right? :rofl: When you have VID:PID identifiers... one would think that was a better way to identify a device :facepalm: Then again, I could be judging them unfairly... maybe it is Sony that is to blame? :shrug: